crowbar_framework/config/initializers/crowbar_checks.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "crowbar/checks"
+
+unless ENV["SKIP_CHECKS"] && ENV["SKIP_CHECKS"] == "yes"
+  network_checks = Crowbar::NetworkChecks.new(Rails.logger)
+
+  unless network_checks.run_all_checks
+    raise "Crowbar network checks failed."
+  end
```

Comment:
Hrm, I'm mixed on this approach. I thought we wanted to have the errors appear in the webui, so raising an exception during initialization won't make that possible.

Or am I misunderstanding?

Also, do we want to run the checks on each startup, or only when we'll enter the wizard mode?